### PR TITLE
Add more readline shortcuts and make ^U work like readline

### PR DESCRIPTION
**Pull request by fratajczak**

### CODE REVIEW

File: src/handlers/input.rs [hunk 1]
```diff
@@ -9,11 +9,37 @@ use unicode_width::{UnicodeWidthChar, UnicodeWidthStr};
...
   match key {
+    Key::Ctrl('k') => {
+      app.input.drain(app.input_idx..app.input.len());
+    }
     Key::Ctrl('u') => {
+      app.input.drain(..app.input_idx);
+      app.input_idx = 0;
+      app.input_cursor_position = 0;
+    }
+    Key::Ctrl('c') => {
```
Comment (Rigellute) [inline src/handlers/input.rs:20]: `Ctrl('c')` is the global exit key of the program (used [here](https://github.com/Rigellute/spotify-tui/blob/master/src/util.rs#L24)).

What other keybind would be appropriate here instead?

**Comment by fratajczak** [comment]
Maybe Ctrl-L (Clear screen in readline) ?

**Comment by Rigellute** [comment]
`Ctrl-l` sounds good 👍

**Comment by fratajczak** [comment]
I just changed it, should be OK now!

**Comment by Rigellute** [comment]
@all-contributors please add @fratajczak for code

**Pull request merged**